client/my-sites/plugins/controller.js

```diff
@@ -144,6 +145,35 @@ function renderProvisionPlugins() {
 }
 
 controller = {
+	validateFilters: function( filter, context, next ) {
+		const wpcomFilters = [ 'standard', 'business', 'premium' ];
+		const siteUrl = route.getSiteFragment( context.path );
+		const site = sites.getSelectedSite();
+		filter || ( filter = context.params.plugin );
```

Comment:
I don't think I've seen this pattern anywhere else in Calypso. Maybe it would be worth expanding a bit for uniformity's sake

```js
if ( ! filter ) {
	filter = context.params.pugin;
}
```
or
```js
function( chosenFilter, context, next ) {
	…
	const filter = chosenFilter
		? chosenFilter
		: context.params.plugin;
}
```

or something else explicit.